libs/challenge-registry/user-profile/src/lib/user-profile-starred/user-profile-starred.component.html

```diff
@@ -18,12 +18,22 @@ <h3>Starred</h3>
       </button>
     </div>
     <ng-container [ngSwitch]="activeTab">
-      <div *ngSwitchCase="tabs['challenges']" class="card-group">
-        <challenge-registry-challenge-card
-          *ngFor="let challenge of challenges"
-          [challenge]="challenge"
-        >
-        </challenge-registry-challenge-card>
+      <div *ngSwitchCase="tabs['challenges']">
```

Comment:
I think it might be better to place the paginator at the bottom?  That way, as users scroll through the list of cards, they can change the page once they reach the tail end rather than have to scroll back up.